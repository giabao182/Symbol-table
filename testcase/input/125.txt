BEGIN
INSERT x number
END
ASSIGN x 5