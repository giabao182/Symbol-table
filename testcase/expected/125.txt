success
Undeclared: ASSIGN x 5